catchup/peerSelector.go

```diff
@@ -405,7 +424,7 @@ func (ps *peerSelector) addToPool(peer network.Peer, rank int, class peerClass,
 }
 
 // sort the pools array in an ascending order according to the rank of each pool.
-func (ps *peerSelector) sort() {
+func (ps *PeerSelector) sort() {
```

Comment:
You don't need to export this function. Do you?

Reply:
nope